apps/dashboard/src/components/side-navigation/side-navigation.tsx

```diff
@@ -36,10 +39,10 @@ const NavigationGroup = ({ children, label }: { children: ReactNode; label?: str
 export const SideNavigation = () => {
   const { subscription, daysLeft, isLoading: isLoadingSubscription } = useFetchSubscription();
   const isFreeTrialActive = subscription?.trial.isActive || subscription?.hasPaymentMethod;
+  const isEnvironmentManagementEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_ENVIRONMENT_MANAGEMENT_ENABLED);
```

Comment:
Having this as a feature flag is fine, but based on the API implementation, it seems this API will only be supported for managed organizations. What will happen once we enable this feature?  

Will all users be able to interact with it but receive 4XX errors?

Reply:
What do you mean by managed organizations? This is only available to business tier orgs, non business tier orgs will see this page, but the button will have a tooltip stating that to create more environments you need to upgrade

Reply:
by managed organizations, i meant business tiers.
and thanks you answered my question thanks!! :)